ui-core/src/surveyUtils.ts

```diff
@@ -14,22 +17,24 @@ function getFormQuestionsHelper(element: FormElement): FormElement[] {
     : [element]
 }
 
-/** transform the stored survey representation into what SurveyJS expects */
-export function extractSurveyContent(form: VersionedForm) {
-  const parsedSurvey = JSON.parse(form.content) as FormContent
-  const questionTemplates = parsedSurvey.questionTemplates
+const applyDefaultSurveyConfig = (surveyModel: SurveyModel): void => {
+  surveyModel.focusFirstQuestionAutomatic = false
+  surveyModel.showTitle = false
+  surveyModel.widthMode = 'static'
+}
+
+/** Create a SurveyJS SurveyModel from a Juniper FormContent object. */
+export const surveyJSModelFromFormContent = (formContent: FormContent): SurveyModel => {
+  const formContentClone = cloneDeep(formContent)
```

Comment:
Why does this need to be cloned?

Reply:
To avoid modifying the `formContent` object passed in. Line 42 copies properties from the question template to the question.